prisma/schema.prisma

```diff
@@ -112,3 +113,23 @@ model Attendees {
   attendeeId Int
   attendee   Users      @relation(fields: [attendeeId], references: [id])
 }
+
+enum TokenType {
+  GOOGLECAL
+  OUTLOOKCAL
+}
+
+model AccessToken {
+  id              Int       @id @default(autoincrement())
+  user            Users     @relation(fields: [userId], references: [id])
+  userId          Int
+  calendarType    TokenType
```

Comment:
What is the difference in `calendarType` and `tokenType`?
Can we just rename `calendarType` as `tokenType` in case both are the same?

Reply:
`calendarType` can be `Google Calendar / Outlook` whereas `tokenType` can be `bearer token` which we get from google cal/outlook integration.

I don't think merging them together would be a good idea.

Reply:
👍